extensions/http/01_http.js

```diff
@@ -321,7 +324,14 @@
       );
     }
 
-    if (request.headers.get("connection")?.toLowerCase() !== "upgrade") {
+    const connection = request.headers.get("connection");
+    const connectionHasUpgradeOption = connection !== null &&
+      ArrayPrototypeSome(
+        StringPrototypeSplit(connection, ","),
```

Comment:
Actually using a regex of `/\s*,\s*/` here would be better and removes the need for trim.

Reply:
Yeah that is definitely better, I didn't really wanted to make the whole check one "big" regex, but I forgot split takes one as well